### ISSUE TITLE
Provide minimal tests and instructions

### ISSUE THREAD

**Issue by alex-konovalov** [vmjanik/artwarp#5]
To start with, we need to have some minimal test files and instructions how to run them.

**Comment by alex-konovalov** [comment]
For example, this can be placed on Wiki here https://github.com/vmjanik/artwarp/wiki

**Comment by alex-konovalov** [comment]
Wiki now is at https://github.com/dolphin-acoustics-vip/artwarp/wiki and has some initial instructions based on an earlier text by @RohanK22

**Comment by olexandr-konovalov** [comment]
@vmjanik are there any `.ctr` files that we can put here as a demo? I think that may eventually be useful if we want to run automated tests as suggested in #11.

**Comment by vmjanik** [comment]
Yes, definitely. Let's find some good examples from the VIP. This is a task for the biology students. Will mention it in the next session. Also, I foud a really old version of ARTWARP. Are you still "collecting" those?

**Comment by olexandr-konovalov** [comment]
Alternative (for testing) could be to have this files downloadable from somewhere during the test run - but that may be less sustainable than placing a `demo` folder in the repository.

I think at this point old versions are hard to insert into the repository - the process described at https://github.com/dolphin-acoustics-vip/artwarp/wiki/Restoring-the-history was meant to be run once. I am keeping the three older versions used for it "for the history", so if you pass to me another one, can put it next to them :) Just in case we might need to track something back.

**Comment by KaiKerlaff** [comment]
@olexandr-konovalov is this resolved by automated testing added in #87? I am trying to clean up this repo's issues page.

**Comment by olexandr-konovalov** [comment]
Those are automated tests. What I think is remaining is to update https://github.com/dolphin-acoustics-vip/artwarp/wiki after the release of v2.0 with CLI support, explaining both GUI and command line manual testing.

**Comment by olexandr-konovalov** [comment]
P.S. even better - update README, not Wiki, to keep specific instructions close to the code. @tamrachan started that in #106 already for the CLI part.
